core/lib/zksync_core/src/lib.rs

```diff
@@ -230,6 +233,8 @@ pub enum Component {
     Housekeeper,
     /// Component for exposing APIs to prover for providing proof generation data and accepting proofs.
     ProofDataHandler,
+    /// Native ERC20 fetcher
+    NativeERC20Fetcher,
```

Comment:
Currenty the API component runs with a gas adjuster, so it's not really possible to run API without the fetcher, because the gas adjuster has (a reference to) the fetcher.

Reply:
I will close it as it was resolved [here](https://github.com/lambdaclass/zksync-era/issues/192)